locale/de/translations.json

```diff
@@ -479,7 +479,6 @@
   "Drafts are not synced to other devices and browsers.": "Entwürfe werden nicht mit anderen Geräten und Browsern synchronisiert.",
   "Drafts from conversation with {recipient}": "Entwürfe aus der Unterhaltung mit {recipient}",
   "Drafts from {recipient}": "Entwürfe von {recipient}",
-  "Drafts older than <strong>{draft_lifetime}</strong> days are automatically removed.": "Entwürfe, die älter als <strong>{draft_lifetime}</strong> Tage sind, werden automatisch entfernt.",
```

Comment:
Don't edit translation files; see
https://zulip.readthedocs.io/en/latest/translating/internationalization.html for details.